### PR TITLE
Update Automation strings

### PR DESCRIPTION
**Pull request by KevinFCormier**
https://github.com/stolostron/backlog/issues/24555

### CODE REVIEW

File: frontend/public/locales/en/translation.json [hunk 1]
```diff
@@ -2309,7 +2309,6 @@
     "template.modal.linkProvider.submit": "Link",
     "template.modal.linkProvider.submitting": "Linking",
     "template.modal.linkProvider.title": "Link an Ansible connection",
-    "template.modal.name.helper.text": "The Ansible job template name must match the name of the job on the Ansible Tower. If the names are not the same, the hook cannot run the job.",
```
Comment (KevinFCormier) [inline frontend/public/locales/en/translation.json:2312]: The user cannot enter their own name here - if it doesn't match one we fetched from the server, it is cleared. So this text does not make sense.

File: frontend/src/routes/Infrastructure/Automations/AnsibleAutomationsForm.tsx [hunk 2]
```diff
@@ -566,7 +569,6 @@ function EditAnsibleJobModal(props: {
                     label={t('template.modal.name.label')}
                     id="job-name"
                     value={ansibleJob?.name}
-                    helperText={t('template.modal.name.helper.text')}
```
Comment (KevinFCormier) [inline frontend/src/routes/Infrastructure/Automations/AnsibleAutomationsForm.tsx:569]: The user cannot enter their own name here - if it doesn't match one we fetched from the server, it is cleared. So this text does not make sense.

File: frontend/public/locales/en/translation.json [hunk 3]
```diff
@@ -2294,7 +2294,7 @@
     "template.emptyStateMsg": "Click <bold>Create Ansible template</bold> to create your resource.",
     "template.hint": "Create an Ansible automation job template to automatically run Ansible jobs at different stages of the cluster life cycle. To use this feature, the Ansible Automation Platform Resource Operator must be installed. Install the operator through the following link:",
     "template.hint.noConsoleURL": "Create an Ansible automation job template to automatically run Ansible jobs at different stages of the cluster life cycle. To use this feature, the Ansible Automation Platform Resource Operator must be installed. Install the operator through OperatorHub.",
-    "template.information.description": "The default job templates that you select appear automatically during cluster creation. To create a sequence of events, select multiple jobs. Drag and drop the job templates to reorder the sequence.",
+    "template.information.description": "To create a sequence of events, add multiple jobs. Drag and drop the job templates to reorder the sequence.",
```
Comment (KevinFCormier) [inline frontend/public/locales/en/translation.json:2297]: First sentence did not convey anything meaningful. And the rest of the text applies better to the Install and Upgrade steps.

**Review by RobDolares — APPROVED** [review]
/lgtm

**Pull request merged**